playbooks/preflight-check.yml

```diff
@@ -0,0 +1,64 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Preflight check
+  hosts: "all_containers"
+  gather_facts: false
+  user: root
+  tasks:
+    - name: Ping check
+      ping:
+      register: ping_result
+      failed_when: "'FAILED' in ping_result"
```

Comment:
This will already fail if the task fails.

Reply:
OK, no need to register response variable.